### PR TITLE
Feature [MGX-251] - Default transactional

### PR DESCRIPTION
**Pull request by iStrike7**
To be merged with https://github.com/mangata-finance/substrate/pull/75

### CODE REVIEW

File: pallets/xyk/src/tests.rs [hunk 1]
```diff
@@ -1198,6 +1198,7 @@ fn sell_N_not_enough_selling_assset() {
 }
 
 #[test]
+#[ignore]
```
Comment (mateuszaaa) [inline pallets/xyk/src/tests.rs:1201]: cant we align those tests instead of disabling them

**Review by mateuszaaa — CHANGES_REQUESTED** [review]
could you provide some UT that tests new impl ?

File: pallets/xyk/src/lib.rs [hunk 2]
```diff
@@ -2303,18 +2314,36 @@ impl<T: Config> XykFunctionsTrait<T::AccountId> for Pallet<T> {
...
+					bought_asset_amount,
+					min_amount_out,
+				));
+			}
 		}
 
 		Ok(())
 	}
 
+	// To put it comprehensively the only reason that the user should lose out on swap fee
```
Comment (majusko) [inline pallets/xyk/src/lib.rs:2334]: Not sure if achievable, but would it be possible to align this comment with autogenerated RustDoc format we are currently using? cc: @mateuszaaa

File: runtime/mangata-kusama/src/lib.rs [hunk 3]
```diff
@@ -160,11 +160,11 @@ impl_opaque_keys! {
 pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("mangata-parachain"),
 	impl_name: create_runtime_str!("mangata-parachain"),
-	authoring_version: 10,
-	spec_version: 10,
+	authoring_version: 11,
```
Comment (majusko) [inline runtime/mangata-kusama/src/lib.rs:163]: This should be incremented only when preparing the RC, so we don't have conflicts.

**Review by majusko — CHANGES_REQUESTED** [review]

File: runtime/mangata-kusama/src/lib.rs (hunk 3 above)
Comment (iStrike7) [reply runtime/mangata-kusama/src/lib.rs:163]: We need to do this, so that it doesn't fail ci.

Comment (majusko) [reply runtime/mangata-kusama/src/lib.rs:163]: Understand. Didn't know. Noting it down for some future CI improvements.

File: pallets/xyk/src/lib.rs (hunk 2 above)
Comment (majusko) [reply pallets/xyk/src/lib.rs:2334]: Ok, @mateuszaaa explained that it would be there like this. So ignore my comment.

**Review by mateuszaaa — APPROVED** [review]
lgtm

**Pull request merged**